src/main/java/io/shodo/BasketInformations.java

```diff
@@ -0,0 +1,56 @@
+package io.shodo;
+
+import java.util.HashMap;
+
+/**
+ * Created by thomas on 02/12/2019.
+ * This class is a basket
+ */
+public class BasketInformations {
```

Comment:
```suggestion
public class Basket {
```

You should have a more self-explanatory name